model/faas/entities.yaml

```diff
@@ -1,14 +1,18 @@
 groups:
-  - id: entity.faas
+  - id: entity.faas.function
     type: entity
     stability: development
-    name: faas
+    name: faas.function
     brief: >
-      A serverless instance.
+      A serverless function.
     attributes:
+      - ref: faas.function.id
+        role: identifying
+        requirement_level: required
```

Comment:
While semantically desirable, in practice we cannot require this as resource attribute I believe, see https://github.com/open-telemetry/opentelemetry-specification/issues/1261#issuecomment-1198285795

Reply:
Why can't we use cloud.resource_id for that consistent identity? It seems you think cloud.resource_id is AWS-specific? That is not the case.